packages/nimble-components/src/table-column/base/tests/table-column.fixtures.ts

```diff
@@ -58,3 +60,59 @@ export class TableColumnDelegatesClickAndKeydown extends TableColumn {
         });
     }
 }
+
+/**
+ * Column validator used by TableColumnValidationTest
+ */
+export class TestColumnValidator extends ColumnValidator<
+readonly ['invalidFoo', 'invalidBar']
```

Comment:
Having to duplicate the array of flag names here and in the constructor is not ideal. But it seems to be the only way we can get the strict typing we're after.

Reply:
Guessing you can do something like:

```ts
const configValidity = ['invalidFoo', 'invalidBar'] as const;
export class TestColumnValidator extends ColumnValidator<
typeof configValidity
> {
    public constructor(columnInternals: ColumnInternals<unknown>) {
        super(columnInternals, configValidity);
    }

    public validateFoo(isValid: boolean): void {
        this.setConditionValue('invalidFoo', !isValid);
    }

    public validateBar(isValid: boolean): void {
        this.setConditionValue('badNameShouldBeBroken', !isValid);
    }
}
```